adm/src/config.rs

```diff
@@ -24,7 +24,8 @@ const DEFAULT_DATA_DIR: &str = "/var/lib/sawtooth";
 const DEFAULT_KEY_DIR: &str = "/etc/sawtooth/keys";
 const DEFAULT_POLICY_DIR: &str = "/etc/sawtooth/policy";
 
-const DEFAULT_BLOCKSTORE_FILENAME: &str = "block-00.lmdb";
+const DEFAULT_BLOCKSTORE_FILENAME_LMDB: &str = "block-00.lmdb";
+const DEFAULT_BLOCKSTORE_FILENAME_ROCKSDB: &str = "block-00.rocksdb";
```

Comment:
The admin tool appears to have no options for the blockstore name, so this is just a convention based on paths from sawtooth env vars?